spring-cloud-gcp-logging/src/test/java/com/google/cloud/spring/logging/LoggingAppenderTests.java

```diff
@@ -65,32 +62,14 @@ void testGetLoggingOptions() {
         .isNotNull()
         .contains("spring-cloud-gcp-logging")
         .contains("Spring");
-
-    loggingAppender.start();
-    // java-logging-logback appender default is ERROR
-    verify(logging).setFlushSeverity(com.google.cloud.logging.Severity.ERROR);
-  }
-
-  @Test
-  void testDisablingFlushLevel() {
-    LoggingAppender loggingAppender = new TestLoggingAppender();
-    loggingAppender.setLogDestinationProjectId("my-log-destination-project");
-    loggingAppender.setFlushLevel(Level.OFF);
-
-    loggingAppender.start();
-    verify(logging, never()).setFlushSeverity(any());
```

Comment:
since we are removing all occurrences `verify(logging)`, I believe we can delete the TestLoggingAppender class and revert back to use the base class for tests in this class. Basically reverting most of changes made to this class from https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/2922/files#diff-cd11f343096de66a41d821803b737c68495d3e1f52f1c86dfe153b4fa8daba06

Reply:
Removed